### ISSUE TITLE
Avaialble new JsonSchema draft-07 

### ISSUE THREAD

**Issue by tincumagic** [noirbizarre/flask-restplus#556]
There is a new version of jsonschema 3.0.0a3 that is able to validate json schema draft7 please update the flask-restplus to use it. 

I have tested in my project and it works even better that the draft 4 because it was able to validate json schema include in other json schema with definitions. The change is small in \flask_resplus\model.py and \flask_restplus\schemas\__init__.py you need to replace: Draft4Validator to Draft7Validator

Together with this fix: https://github.com/tincumagic/flask-restplus/commit/469e4c063f5362ef16a9ebc505e1223083e2bc17 it allows do document and correctly validate nested jsonschema in #definitions.

**Comment by louispotok** [comment]
Agreed, it would be nice to see this (or, dare I dream, the ability to specify which version of the jsonschema draft you want to use!). In the meanwhile, the following hack worked when put above my `Api` declaration.

```
from jsonschema.validators import Draft7Validator
import flask_restplus.model

flask_restplus.model.Draft4Validator = Draft7Validator```
